src/libostree/ostree-deployment.c

```diff
@@ -87,24 +111,49 @@ ostree_deployment_get_origin (OstreeDeployment *self)
   return self->origin;
 }
 
+/**
+ * ostree_deployment_get_index:
+ * @self: Deployment
+ *
+ * Returns: The global index into the bootloader ordering
+ */
 int
 ostree_deployment_get_index (OstreeDeployment *self)
 {
   return self->index;
 }
 
+/**
+ * ostree_deployment_set_index:
+ * @self: Deployment
+ *
+ * Sets the global index into the bootloader ordering.
+ */
 void
 ostree_deployment_set_index (OstreeDeployment *self, int index)
 {
   self->index = index;
 }
 
+/**
+ * ostree_deployment_set_bootserial:
+ * @self: Deployment
+ *
+ * Should never have been made public API; don't use this.
+ */
 void
 ostree_deployment_set_bootserial (OstreeDeployment *self, int index)
 {
   self->bootserial = index;
 }
 
+/**
+ * ostree_deployment_set_bootconfig:
+ * @self: Deployment
+ * @bootconfig: (nullable): Set the bootloader options for this deployment
```

Comment:
This looks like a description of the API rather than of the argument.

Maybe just

```suggestion
 * @bootconfig: (nullable): Bootloader config object
```